server/metadata/cluster_test.go

```diff
@@ -22,17 +22,16 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/tigrisdata/tigris/errors"
+	"github.com/tigrisdata/tigris/lib/uuid"
 	"github.com/tigrisdata/tigris/server/transaction"
 )
 
 var testClusterPayload = &ClusterMetadata{
-	WorkerKeepalive: time.Now().UTC(),
+	ID: uuid.New(),
```

Comment:
What do we use the cluster id for? And what do we plan to use it for in the future?